pkg/cri/server/container_status.go

```diff
@@ -120,6 +121,44 @@ func toCRIContainerStatus(container containerstore.Container, spec *runtime.Imag
 		Mounts:      meta.Config.GetMounts(),
 		LogPath:     meta.LogPath,
 	}
+
+	if status.Resources != nil && status.Resources.Linux != nil {
```

Comment:
per discussion let's make a copy of the status in the container store get under the mutex..

Reply:
Done, PTAL